### DISCUSSION TITLE
Will Soon be Adding Level Labels!

### DISCUSSION THREAD

**Discussion by MonalikaPatnaik** [MonalikaPatnaik/UMatter#428] [Announcements]
Hey, we will soon be adding level labels to the issues but remember, we will add labels to your issue once your PR is merged!
So, just don't worry...keep up the good work!
Keep contributing...less Go!

**Comment by shivarm** [comment]
3 days before PR which got merged they don't have level

**Comment by shivarm** [comment]
level help us to score++ right?

**Comment by Piyusharora2003** [comment]
Created an entire page fully responsive with data getting from external source ( specific blog page ) with customizable image input , 
Fixed an issue in the generic blog page ,
Still got level 1 😔 

To verify - #711
